### ISSUE TITLE
Enable single point without Boltzmann weighting

### ISSUE THREAD

**Issue by danielhollas** [ispg-group/aiidalab-ispg#47]
Normal workflow is to have a set of conformers, optimize them and do Wigner sampling.

For quick estimates, or for high level theory calculation, we also want to provide an option of doing only single point calculations on the set of preoptimized conformers. The resulting spectrum would still be Boltzmann weighted of course (although without optimization, does it make sense? Maybe we should always optimize, but only perhaps skip Wigner and maybe live dangerously and skip frequencies as well. All configurable in advanced options. (Btw: rename Expert mode to Advance options).

As an additional option, we could skip the Boltzmann weighting entirely and assume that the user already provided statistically samples structures. This would allow us to easily calculate spectra from ab initio trajectories.

Blocked on #48

**Comment by danielhollas** [comment]
Using single point calculations without Boltzmann weighting currently works, anything else is out of scope for now.
